integration_tests/utils.py

```diff
@@ -35,6 +35,20 @@
 KEYS = {name: account.key for name, account in ACCOUNTS.items()}
 ADDRS = {name: account.address for name, account in ACCOUNTS.items()}
 CRONOS_ADDRESS_PREFIX = "crc"
+CONTRACTS = {
+    "Gravity": Path(__file__).parent
+    / "contracts/artifacts/contracts/Gravity.sol/Gravity.json",
+    "Greeter": Path(__file__).parent
+    / "contracts/artifacts/contracts/Greeter.sol/Greeter.json",
+    "ModuleCRC20": Path(__file__).parent.parent
+    / "x/cronos/types/contracts/ModuleCRC20.json",
+    "TestERC20A": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestERC20A.sol/TestERC20A.json",
+    "TestRevert": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestRevert.sol/TestRevert.json",
+    "TestERC20Utility": Path(__file__).parent
+    / "contracts/artifacts/contracts/TestERC20Utility.sol/TestERC20Utility.json",
```

Comment:
share the `Path(__file__).parent / "contracts/artifacts/contracts"` with a global var?

Reply:
Do you mean
```
TEST_CONTRACT_PATH = Path(__file__).parent / "contracts/artifacts/contracts"
CONTRACTS = {
    "Gravity": TEST_CONTRACT_PATH / "Gravity.sol/Gravity.json",
    "Greeter": TEST_CONTRACT_PATH / "Greeter.sol/Greeter.json",
    "ModuleCRC20": Path(__file__).parent.parent
    / "x/cronos/types/contracts/ModuleCRC20.json",
    "TestERC20A": TEST_CONTRACT_PATH / "TestERC20A.sol/TestERC20A.json",
    "TestRevert": TEST_CONTRACT_PATH / "TestRevert.sol/TestRevert.json",
    "TestERC20Utility": TEST_CONTRACT_PATH
    / "TestERC20Utility.sol/TestERC20Utility.json",
}

```

Reply:
Or just one `TEST_CONTRACT_PATH = Path(__file__).parent / "contracts/artifacts/contracts"`?